app-react/src/components/TransactionView/types.ts

```diff
@@ -0,0 +1,12 @@
+import { Account, Transaction } from '../../../../app-express/src/api/types';
+
+export interface TransactionViewProps {
+  transaction: Transaction;
+  index: number;
+  account?: Account;
+}
+
+export enum TransactionType {
```

Comment:
I duplicated the types that resides in the folder of the server, just to make my life easier